src/styles/variables.scss

```
@@ -1,4 +1,29 @@
 // ----------------------------------------------------
+// 缩放系数
+:root {
+  --nut-scale-f: 1;
+  --nut-scale-font: var(--nut-scale-f, 1);
+  --nut-scale-icon: var(--nut-scale-f, 1);
+  // ---- icon size ----
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)